### ISSUE TITLE
Distinct avy faces

### ISSUE THREAD

**Issue by d125q** [hlissner/emacs-doom-themes#178]
Currently, all `avy` faces are the same.  This makes some operations completely unusable.

Consider:

```elisp
;; Display time, load, and mail indicators in the mode line.
(use-package time
  :custom
  (display-time-day-and-date t)
  (display-time-24hr-format t)
  (display-time-mode t))
```

Doing `M-x avy-goto-char <RET> d` gives:

![avy with doom-one](https://user-images.githubusercontent.com/7765538/39629738-771046d2-4fad-11e8-8367-73581e5b92a0.png)

This is utterly confusing as there is no way to distinguish which character is first and which is second (or possibly third).

**Comment by hlissner** [comment]
You're right. Not enough thought went into avy's faces. They've been fixed as of 410d4ae. It may take a while for MELPA to pick up the changes, but please let me know if the contrast is sufficient.

**Comment by d125q** [comment]
Looks pretty good to me; thanks!
